web-gateway/app/views/createItem.scala.html

```diff
@@ -10,6 +10,7 @@
 
     @foundationForm(itemForm, routes.ItemController.createItem()) {
         <div class="column row">
+            <P>@message("instruction.createItem")</P>
```

Comment:
It'd be good to stick to a single tag capitalization approach. There's some `<P>` and other `<p>`. I think lowercase should be maintained.